libr/debug/p/native/linux/linux_debug.c

```diff
@@ -223,11 +233,17 @@ RDebugReasonType linux_ptrace_event (RDebug *dbg, int pid, int status) {
 			r_sys_perror ("ptrace GETEVENTMSG");
 			return R_DEBUG_REASON_ERROR;
 		}
-		linux_add_and_attach_new_thread (dbg, (int)data);
+		linux_add_new_thread (dbg, (int)data);
 		if (dbg->trace_clone) {
 			r_debug_select (dbg, dbg->pid, (int)data);
 		}
 		eprintf ("(%d) Created thread %d\n", dbg->pid, (int)data);
+		// Continue the parent task
```

Comment:
This seems to be duplicated in other cases as well, it should be handled outside of linux_ptrace_event using the debug reason.

Reply:
Because every `fork` and `clone` will stop the parent and its child together, I will `PTRACE_CONT` one of them based on the debugger option `dbg->trace_clone` and `dbg->trace_fork` and another will be assigned to `dbg->tid` to be handled outside.